configuration/config-txt/video.md

```diff
@@ -483,27 +483,31 @@ Setting `hdmi_ignore_hotplug` to `1` pretends that the HDMI hotplug signal is no
 
 ### disable_overscan
 
-Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan).
+Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan). The default value is `0`.
+
+Disabling overscan is accomplished by setting `overscan_left`, `overscan_right`, `overscan_top` and `overscan_bottom` together to `0`. Any further additional overscan setting such as `overscan_scale` can still be applied.
 
 ### overscan_left
 
-The `overscan_left` command specifies the number of pixels to skip on the left edge of the screen. Increase this value if the text flows off the left edge of the screen; decrease it if there is a black border between the left edge of the screen and the text.
+The `overscan_left` command specifies the number of pixels to skip on the left edge of the screen. Increase this value if the text flows off the left edge of the screen; decrease it if there is a black border between the left edge of the screen and the text. The default value is `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.
 
 ### overscan_right
 
-The `overscan_right` command specifies the number of pixels to skip on the right edge of the screen.
+The `overscan_right` command specifies the number of pixels to skip on the right edge of the screen. The default value is `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.
 
 ### overscan_top
 
-The `overscan_top` command specifies the number of pixels to skip on the top edge of the screen.
+The `overscan_top` command specifies the number of pixels to skip on the top edge of the screen. The default value is `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.
 
 ### overscan_bottom
 
-The `overscan_bottom` command specifies the number of pixels to skip on the bottom edge of the screen.
+The `overscan_bottom` command specifies the number of pixels to skip on the bottom edge of the screen. The default value is `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.
 
 ### overscan_scale
 
-Set `overscan_scale` to `1` to force any non-framebuffer layers to conform to the overscan settings.
+Set `overscan_scale` to `1` to force any non-framebuffer layers to conform to the overscan settings. The default value is `0`.
+
+Note: this setting is generally not recommended because it can reduce performance as the scaling is done by the GPU.
```

Comment:
Performance isn't affected but quality is as all layers on the display will be scaled. Disabling overscan on the TV is the recommended options to avoid images being scaled twice (by Pi and TV).